purchases-js-hybrid-mappings/src/purchases-common.ts

```diff
@@ -40,15 +41,33 @@ export class PurchasesCommon {
       flavor: configuration.flavor,
       version: configuration.flavorVersion,
     });
+
+    let appUserId: string;
+    if (configuration.appUserId !== undefined) {
+      appUserId = configuration.appUserId;
+      if (typeof window !== 'undefined') {
```

Comment:
Why don't we check for the existence of `localStorage` rather than `window`?

Reply:
That's a very good point 😅 changed in https://github.com/RevenueCat/purchases-hybrid-common/pull/1147/commits/b8bbb77841cdcfcc82f2db736250eb62c27d68f5